tests/topotests/lib/common_config.py

```diff
@@ -1258,6 +1261,19 @@ def verify_rib(tgen, addr_type, dut, input_dict, next_hop=None, protocol=None):
     return True
 
 
+def verify_rib(tgen, addr_type, dut, input_dict, next_hop=None, protocol=None):
+    "Wrapper function for `_verify_rib` that tries multiple time to get results."
+
+    # Use currying to hide the parameters and create a test function.
+    test_func = partial(_verify_rib, tgen, addr_type, dut, input_dict, next_hop, protocol)
```

Comment:
If this is the only place it's used can't we just put `_verify_rib` inside this function and make it a closure?
This isn't a blocking comment, just wondering if that might be a bit cleaner.